### DISCUSSION TITLE
Version 1.0 Changes/Suggestions

### DISCUSSION THREAD

**Discussion by grantshandy** [skylinecc/fdg#17] [Announcements]
@micahscopes @yberreby

You all expressed interest in contributing/waiting for 1.0 in #15  and #16, I've pushed an MVP of what I think the API should look like for 1.0 onto the `main` branch of the repo. Any suggestions/improvements before I push to crates.io?

@yberreby, if you want to experiment with SIMD/Parallelization/GPU compute I can definitely hold off on releasing it if you want your work in 1.0. I know these issues were brought up six months ago, but if you're still interested in contributing I'd love to facilitate that. If not, no pressure, I am gonna explore those routes on my own too 😄.

**Comment by yberreby** [comment]
Hi @grantshandy, very sorry for taking so long to respond! I've been quite swamped with research commitments, which has pushed back my intention to spend some time contributing to this library. Glad to see you've put out 1.0!

For context, my intention was/is to use `fdg` for a side project of mine that requires very large graphs to be rendered. Hopefully, when I can find the bandwidth for that once again, I'd be able to make some performance contributions.

**Reply by grantshandy** [reply]
No pressure at all 👍🏽. Feel free to look into it or contribute whenever works for you.
